symja_android_library/matheclipse-core/src/main/java/org/matheclipse/core/expression/HMArrayList.java

```diff
@@ -94,9 +94,6 @@
     firstIndex = hashValue = 0;
     lastIndex = arguments.length + 1;
     switch (lastIndex) {
-      case 0:
```

Comment:
This case can never happen because `arguments.length` is at least zero (never negative) and plus 1 makes the lowest possible value one.